polar-core/src/debugger.rs

```diff
@@ -292,12 +293,20 @@ impl Debugger {
                 if parts.len() > 1 {
                     let vars: Vec<Binding> = parts[1..]
                         .iter()
-                        .map(|var| {
-                            let var = Symbol::new(var);
-                            let value = vm.bindings(true).get(&var).cloned().unwrap_or_else(|| {
-                                Term::new_temporary(Value::Variable(Symbol::new("<unbound>")))
-                            });
-                            Binding(var, value)
+                        .map(|nom| {
+                            let var = Symbol::new(nom);
+                            let bindings = vm.bindings(true);
+                            bindings.get(&var).cloned().map_or_else(|| {
+                                let pref = KnowledgeBase::temp_prefix(nom);
```

Comment:
Lengthening this to `prefix` would help me too.

Reply:
👍